### ISSUE TITLE
Kalenderansicht der Buchung evtl. problematisch bei zu vielen Buchungen?

### ISSUE THREAD

**Issue by mimosel** [tiloman/simren-issues#88]
Ich habe den Verdacht, der vielleicht falsch ist, dass die Kalenderansicht 

![grafik](https://github.com/tiloman/simren-issues/assets/129038745/fa402d17-08fc-49d6-9ae1-b4ac6ea2d547)

problematisch bzw. sehr aufgebläht oder in die Höhe gezogen werden könnte, bei vielen gleichzeitigen Ausleihen. Wir haben z.T. 30 Ausleihen parallel laufen über viele verschiedene Tage. Ich befürchte so ein bisschen, dass das die Kalenderansicht massiv aufblähen könnte und unübersichtlich machen könnte...?

**Comment by tiloman** [comment]
Ja, die Kalenderübersicht eignet sich für die Ansicht aller Buchungen nicht und ich überlege diese Ansicht komplett auszuschalten. Für einzelne Ressourcen mag das Sinn ergeben, aber nicht für alle gleichzeitig.

**Comment by mimosel** [comment]
komplett ausschalten fände ich sehr unpraktisch. Wir haben viele Ausleihen parallel und würden gerne den Überblick behalten. Vielleicht könnte man die Übersicht einfach auf fünf limitieren und den Rest ausklappbar machen oder so?

**Comment by mimosel** [comment]
Übersichtlichkeit. Hier sieht man halt wirklich alles auf einen Blick..., während man bei den Filtern noch Parameter setzen muss und auch aufpassen muss, das man wirklich die richtigen Parameter gesetzt hat, damit einem nichts entgeht.

**Comment by mimosel** [comment]
Das Thema kam bei uns nochmal hoch. Hier mal zur Verdeutlichung, wie unsere Übersicht im alten System aussah.

![Image](https://github.com/user-attachments/assets/8e899108-8397-4f3e-ae3d-0a2701c65347)

Der Vorteil hier ist, dass man für den bspw. 06.01. direkt mit einem Blick (die Spalte herunter) sieht, was alles an Geräten rausgeht und man auch leicht erfährt, wann welches Gerät wiederkommt.

**Comment by mimosel** [comment]
![Image](https://github.com/user-attachments/assets/ad0d13ae-c523-4420-bca5-68ae430e6563)

Unsere Befürchtung hat sich bestätigt, dass die Kalenderübersicht sehr unübersichtlich ist. Vielleicht könnte es helfen, wenn man die Kalenderansicht - ähnlich unser Excel-Tabelle aus dem letzten Kommentar - in die Breite ziehen würde, also dass jeder Tag eine Spalte ist?
